portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +26,49 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private List<String> comments;
+
+  @Override
+  public void init() {
+    comments = new ArrayList<>();
+
+    // comments.add("josh : I like this site");
+    // comments.add("marie : I was promised cat pictures :( ");
+    // comments.add("steven : I am a student at UoM too! :o ");
```

Comment:
This can be removed, or uncommented so there are some initial comments?

Reply:
Yes, those were there initially when the comments were hard-coded. I forgot to remove them